### PR TITLE
LPS-171974 Replace liferay-ui:icon and aui:icon in view.jsp

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
### Case A

1. Create a new site.
2. Go to Categorization > Categories.
3. Hover the symbols next to the global vocabularies to check the text info. 

image
image

### Case B
In the same site.

1. Create a new vocabulary with internal visibility.
2. Hover the symbols next to the new vocabulary to check the text info. 

image
image

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
image

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -283,7 +282,9 @@
 					<c:if test="<%= assetCategoriesDisplayContext.isAssetCategoriesLimitExceeded() %>">
 						<div class="alert alert-warning">
```
Comment (ealonso) [inline modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/view.jsp:283]: Hey @BarbaraCabrera, 

Why not clay:alert?

**Comment by veroglez** [comment]
see https://github.com/liferay-echo/liferay-portal/pull/10794

**Pull request closed without merging**